src/audio/codec_adapter/codec_adapter.c

```diff
@@ -91,6 +91,12 @@ static struct comp_dev *codec_adapter_new(const struct comp_driver *drv,
 	return NULL;
 }
 
+static inline int validate_setup_config(struct ca_config *cfg)
+{
+	/* TODO: validate codec_adapter setup parameters */
```

Comment:
Can't this end up being pluggable (specific to each codec)?

Reply:
This is to validate the config of SoF component codec_adapter. For codecs there is none validation function at the moment, but sure I will add one :)